### PR TITLE
feat(utils): create a `toPosixPath` method

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Since there are multiple places where slashes in filepaths need to be transformed (Windows => Posix), this PR creates a reusable `toPosixPath` method.

Similar to what is supplied in the [docusaurus utils](https://github.com/facebook/docusaurus/blob/fed25eed0b71d37adb772ddeab8910836b0fb71c/packages/docusaurus-utils/src/pathUtils.ts#L58-L65) except in that version it also checks for extended slashes at the beginning of the string, which I don't think is needed in this project.

The method is applied to places where the path could not be read due to path resolutions in Windows.

## Related Issue
N/A

### CODE REVIEW

File: src/lib/utils/relativePath.ts [hunk 1]
```diff
@@ -2,3 +2,8 @@ import { join } from "path"
 
 export const getRelativePath = (routerPath: string, fileName: string) =>
   join("content", routerPath, fileName)
+
+/**
+ * Overcomes dev environment issue on Windows. Converts windows path to posix path.
```
Comment (TylerAPfledderer) [inline src/lib/utils/relativePath.ts]:
```suggestion
 * Replaces back slashes of file paths generated in Windows OS with Node to forward slashes.
```

File: src/components/MarkdownImage.tsx [hunk 2]
```diff
@@ -22,7 +23,7 @@ const MarkdownImage = ({
   let imageWidth = parseFloat(width)
   let imageHeight = parseFloat(height)
 
-  const srcToString = src.toString()
+  const transformedSrc = toPosixPath(src.toString())
```
Comment (nhsz) [inline src/components/MarkdownImage.tsx:27]: we should add a comment above here explaining rbiefly why is this needed

**Review by nhsz — COMMENTED** [review]
@TylerAPfledderer great, LGTM, just asking for a comment for future users

**Review by pettinarip — APPROVED** [review]
Nice, thanks for the refactor @TylerAPfledderer looks good

File: src/components/MarkdownImage.tsx [hunk 3]
```diff
@@ -22,7 +23,8 @@ const MarkdownImage = ({
   let imageWidth = parseFloat(width)
   let imageHeight = parseFloat(height)
 
-  const srcToString = src.toString()
+  // Ensure that src path has forward slashes only
```
Comment (nhsz) [inline src/components/MarkdownImage.tsx]:
```suggestion
  // Ensure that src path has forward slashes only, to avoid issues with Windows filepaths
```

**Review by nhsz — APPROVED** [review]
